test/e2e/apps/rc.go

```diff
@@ -260,7 +280,7 @@ var _ = SIGDescribe("ReplicationController", func() {
 
 		ginkgo.By("waiting for ReplicationController to have a DELETED watchEvent")
 		for watchEvent := range rcWatchChan {
-			if watchEvent.Type == "DELETED" {
+			if watchEvent.Type == watch.Deleted {
```

Comment:
I have the same concern as https://github.com/kubernetes/kubernetes/pull/89707 that this test will not fail if the watch times out